src/components/configuration/current_working_directory_manager.ts

```diff
@@ -93,7 +102,7 @@ export class CurrentWorkingDirectoryManager {
 
         return this.checkPathExists(pathToCargoTomlInPreviousCwd).then<string>(exists => {
             if (exists) {
-                return Promise.resolve(this.rememberedCwd);
+                return Promise.resolve(<string>this.rememberedCwd);
```

Comment:
What is the change for?

Reply:
This was done so that the linter would stop nagging since there, we say that we will eventually return a `Promise<string>`, but when resolving with `Promise.resolve(this.rememberedCwd)`, the final type of the promise that we are returning is actually `Promise<string|undefined>`.

Thus, to get rid of the inconsistency I wrote it explicitly since at that point we will not return `undefined`.

Reply:
I believe it could be spelled like `this. rememberedCwd!`, just to drop the undefined type. That way you still propagate the string type instead of asserting it.

Reply:
@Frizi, you are correct, I changed that part to `this.rememberedCwd!` as per your comment.